website_multi_theme/data/themes_default.xml

```diff
@@ -15,6 +15,6 @@
 
     <!-- This method prepare assets and applies default theme for all websites -->
     <function model="website.config.settings"
-              name="multi_theme_reload"
+              name="_multi_theme_reload"
```

Comment:
Can't this call an @api.multi method?

Reply:
It raise error. At least in the call without args. It would probably work with ``eval="([],)"``

Reply:
Then I think it's better to try for simplifying the code